s5a/__init__.py

```diff
@@ -51,25 +52,88 @@ def load_ncfile(ncfile):
 
     # convert data to geodataframe
     return pandas.DataFrame({
-            'timestamp': timestamps,
```

Comment:
what do you mena by that? for sure it has something to do with the patch if it fixes the indent

Reply:
Generally, one pull request should fix one thing. If you think indentation needs to be fixed, then create a pull request to fix the indentation. This has nothing to do with `h3` and you claim that to be the topic of your pull request.

In addition, in what way does this *fix* the indentation? According to our testing rules, the previous indentation was perfectly fine since it passes `flake8`.

Reply:
ok i removed it from the actual pr